### ISSUE TITLE
Issue with the proxy server

### ISSUE THREAD

**Issue by WeberMarius** [compas-dev/compas_fea#127]
I work on implementing new mechanical models in Compas fea 1 using Ansys as Backend.
It works really fine, but when I install the packages on another PC (including all required compas/compasFEA packages) the Proxy Server cannot be started (all 99 attempts seem to fail), or the following message appears:

![grafik](https://user-images.githubusercontent.com/49633262/226561201-f1bda328-40fb-4b90-bcb6-2cfd96fd6eb6.png)

Can someone help me out of this issue or give me some advances to solve it?

Thanks a lot

**Comment by franaudo** [comment]
Hi Marius,

did you install `compas_fea` in rhino? 

from the command line try this
```
conda activate name_of_your_environment
python -m compas_rhino.install
python -m compas_rhino.install -p compas_fea
```

Then reset the script engine from the Rhino python editor and try again

**Comment by WeberMarius** [comment]
Yes, compas FEA is installed. The preprocessing and the solver work fine (where I also use compas FEA). But if I use rhino.plot_data, the message posted above occurs.

**Comment by franaudo** [comment]
Can you please share the script you are trying to run and if necessary the rhino model? I will try to replicate the problem on my machine

**Comment by WeberMarius** [comment]
I made several adjustments in compas and compas_fea1 so that the backend (Ansys) and the new structural models work. These changes are not pushed on the main compas, compas_fea1 repositories since the packages are under development, and I have to check them for general functionality. The best is to download compas, compas_fea and the strucenglib_snippets from our fork on https://github.com/StrucEng-Library-kfmresearch. Please use the branch "ansys". Thanks for your support!
